build.gradle

```diff
@@ -2,13 +2,27 @@ group 'com.epam.university'
 version '0.1'
 
 apply plugin: 'java'
+apply plugin: 'checkstyle'
 
 sourceCompatibility = 1.8
 
 repositories {
     mavenCentral()
 }
 
+checkstyle {
+    toolVersion = "8.2"
+}
+
+tasks.withType(Checkstyle) {
+    ignoreFailures = true
+    reports {
+        html.enabled = true
+        xml.enabled = false
+    }
+}
+
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    compileOnly 'org.projectlombok:lombok:1.16.18'
```

Comment:
Do you really need lombok for our tasks?

Reply:
naah..
just addicted)

Reply:
naah..
just addicted)